inc/cloudfront_media_purge/namespace.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Altis\Cloud\Cloudfront_Media_Purge;
+
+use function Altis\Cloud\purge_cdn_paths;
+
+function bootstrap() {
+	add_action( 'delete_attachment', __NAMESPACE__ . '\\purge_media_file_cache', 100 );
+}
+
+/**
+ * Purge an attachment file cache from CloudFront
+ *
+ * It sends a request to invalidate all sizes given for a media file
+ *
+ * @param int $media_id Attachment ID.
+ *
+ * @return bool
+ */
+function purge_media_file_cache( int $media_id ) {
+	$upload_url       = wp_get_attachment_url( $media_id );
+	$upload_path      = wp_parse_url( $upload_url, PHP_URL_PATH );
+	$upload_path_info = pathinfo( $upload_path );
+	$items            = [];
+	$items[]          = $upload_path_info['dirname'] . '/' . $upload_path_info['filename'] . '*';
+	if ( function_exists( 'tachyon_url' ) ) {
+		$tachyon_url       = tachyon_url( wp_get_attachment_url( $media_id ) );
```

Comment:
```suggestion
		$tachyon_url       = tachyon_url( $upload_url );
```
Already have this data so no need to refetch it.